README.md

```diff
@@ -1,2 +1,27 @@
-# ops
-Documentation &amp; project information for the RailsBridge Ops Team
+## Purpose:
+
+The primary purpose of the RailsBridge Ops & IT Team is to ensure the availability, security, and performance of RailsBridge applications and infrastructure.
+
+Secondarily, if team capacity and interests allow for it, it would be great for this team to help teach and share Ops & IT expertise through pairing with other volunteers, authoring relevant curricula, etc.
+
+## How we'll work:
+
+- Work in collaboration with RailsBridge developers, board members, volunteers, and staff, and consult with them on decisions and work that may impact other efforts within the organization.
+- Communicate respectfully, fight bias & descrimination, be open, and welcome people into the club. Relevant resources:
+  - [Hacker School social rules](https://www.hackerschool.com/manual#sec-environment)
+  - [Geek Feminism sample code of conduct](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy)
+  - [Bridget Kromhout's "The First Rule of DevOps Club"](http://bridgetkromhout.com/blog/2014/11/03/the-first-rule-of-devops-club/)
+- Be respectful of team members' time and availability, and work on an opt-in basis. (The "pull request not feature request" work model.)
+
+#### Process 
+
+- Share any Ops/IT issues or potential projects with the team as GitHub issues in the railsbridge/ops repo. Try to use tags like "urgent," "idea," or "question" to make it easier for team members to prioritize issues.
+- If you are working on an urgent issue, post status updates and communicate with other team members in the **bridge-things/railsbridge-ops** Slack channel so others can follow along or catch up on what happened if they join later.
+- If there's an Ops or IT issue that you solve by yourself, post a brief write-up about it as an issue in the ops repo so other team members can know what happened (and learn from how you fixed it!). Include info like how you found out about the problem, what steps you took to diagnose and fix it, and what the outcome of each step was.
```

Comment:
In addition to posting about it after you've solved it, it might be worth encouraging people to open issues _as_ they work through a problem for visibility.